tests/suites/test_suite_psa_crypto_init.data

```diff
@@ -10,6 +10,9 @@ deinit_without_init:0
 PSA deinit twice
 deinit_without_init:1
 
+PSA threaded init checks
+psa_threaded_init:100
```

Comment:
Do we need a `depends_on:MBEDTLS_THREADING_PTHREAD` here?

Reply:
No, this is done on the test function itself, rather than the data. The test cannot run without threading, regardless of what the data is set to.